src/balances/types.ts

```diff
@@ -13,27 +13,35 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import { PortfolioBalanceType } from '../model/enums/PortfolioBalanceType';
-import { VisibilityStatus } from '../model/enums/VisibilityStatus';
-import { GetPortfolioBalancesResponse } from '../model/GetPortfolioBalancesResponse';
-import { GetWalletBalanceResponse as internalGetResp } from '../model/GetWalletBalanceResponse';
-import { ListWeb3WalletBalancesResponse } from '../model/ListWeb3WalletBalancesResponse';
-import { ListEntityBalancesResponse as internalListEntityBalances } from '../model/ListEntityBalancesResponse';
+import { Brand } from 'src/shared/brand';
+import { PortfolioBalanceType, VisibilityStatus } from '../model/enums/';
+import {
+  GetPortfolioBalancesResponse,
+  GetWalletBalanceResponse as internalGetResp,
+  ListWeb3WalletBalancesResponse,
```

Comment:
Rename Web3 to Onchain

Reply:
the openapi spec is still coinbase.public_rest_api.ListWeb3WalletBalancesResponse. changing this would be a breaking change and require a 4.0. Want me to do that?

Reply:
I misread this originally. I am not naming this, this is out of the open api spec. but below I am branding it to the better name of 
export type ListOnchainWalletBalancesResponse = Brand<
  ListWeb3WalletBalancesResponse,
  'ListOnchainWalletBalancesResponse'
>;

The end user will not see the internal Web3 object.